CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+## System Requirements
+
+go 1.18+  is recommended.
+
+## Setup workspace
+ 
+Run the following command to setup the go workspace.
+```
+make workspace-init
+```
+
+sync go.work with current modules
+
+```
+make workspace-update
+```
+
+## Compilation target(s)
```

Comment:
This section is not applicable for go-sdk as we do not build artifacts. Instead, SDK relies on Go lang's source-based dependencies. 

Note - See below example for clarification :) 

```
package main

import (
	"context"
	"fmt"
	"github.com/open-feature/go-sdk-contrib/providers/flagd/pkg"
	"github.com/open-feature/go-sdk/pkg/openfeature"
)

func main() {
	openfeature.SetProvider(flagd.NewProvider())
	client := openfeature.NewClient("app")
	value, _ := client.BooleanValue(
		context.Background(), "v2_enabled", false, openfeature.EvaluationContext{},
	)

	fmt.Printf("flag eval: %t", value)
}

```